unified_planning/test/test_negative_conditions_remover.py

```diff
@@ -99,12 +99,16 @@ def test_matchcellar(self):
         mend_f2 = ActionInstance(mend_fuse, (ObjectExp(f2), ))
         mend_f3 = ActionInstance(mend_fuse, (ObjectExp(f3), ))
         npa = [a for s, a, d in new_plan.actions]
-        self.assertIn(light_m1, npa)
-        self.assertIn(light_m2, npa)
-        self.assertIn(light_m3, npa)
-        self.assertIn(mend_f1, npa)
-        self.assertIn(mend_f2, npa)
-        self.assertIn(mend_f3, npa)
```

Comment:
Maybe (just maybe) we can allow for the `in` operator to work with plans? Something like `action_instance in plan` where `plan` is any kind of plan (in Python this is done by overriding the `__contains__` method)

Reply:
It can be a nice feature, even if not used much. Anyway, I do not see problems in adding it.